commit 600369fe8194f2cf255e66d600dcd09881f5db0f
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-03-21T13:47:23+00:00 (7 months ago)
    
    MANTA-4191 remove deps/node submodule remnants from mola and madtom

diff --git a/.gitmodules b/.gitmodules
index 2717da8..c896b7b 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -7,9 +7,6 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
 	url = https://github.com/davepacheco/javascriptlint.git
-[submodule "deps/node"]
-	path = deps/node
-	url = https://github.com/joyent/node.git
 [submodule "deps/manta-scripts"]
 	path = deps/manta-scripts
 	url = https://github.com/joyent/manta-scripts.git
